lib/autocomplete-paths.js

```diff
@@ -112,6 +112,30 @@ export default {
     this._statusBarElement = null
   },
 
+  _findAndLoadAliasConfig() {
+    if (this._provider) {
+      atom.project.getDirectories().forEach((projDir) => {
+        /* TODO add weppack support
+        var webpackConfig = (configPath && fs.existsSync(configPath)) ? require(configPath) : null;
+        if (typeof webpackConfig === 'function') {
+        webpackConfig = webpackConfig();
+        }
+        */
+        let tsconf = projDir.getFile('tsconfig.json')
+        if (tsconf.existsSync()) {
+          try {
+            tsconf = JSON.parse(tsconf.readSync());
+            if (tsconf.compilerOptions && tsconf.compilerOptions.paths) {
+              this._provider.setAliases(tsconf.compilerOptions.paths);
+            }
+          } catch(e) {
+            console.error('autocomplete-paths: can\'t parse "tsconfig.json" for aliases. incorrect json')
+          }
+        }
+      })
+    }
+  },
```

Comment:
Can this be abstracted away to make aliases general? I suggest making this an option in the scopes.
https://github.com/atom-community/autocomplete-paths/blob/master/lib/config/default-scopes.js#L32


```
aliasReplacer: (projDir) => { // handle alias replacing }
```